src/components/Anchor/styles/sharedStyles.js

```diff
@@ -28,78 +30,83 @@ export const base = css`
   }
 
   &:active {
-    color: ${get('colors.primary.default')};
+    color: ${({ disabled }) =>
+      disabled ? disabledColor : get('colors.primary.default')};
   }
 
-  &::after {
-    content: '';
-    background: ${color};
-    border-radius: 2em;
-    height: 1px;
-    width: 100%;
-    position: absolute;
-    bottom: -0.1em;
-    left: 0;
-    transition: height 0.15s ease, width 0.15s ease, left 0.15s ease,
+  ${({ disabled, appearFocused }) =>
+    disabled
+      ? ''
+      : `
```

Comment:
I'm not sure interpolations consistently work with a plain template string, I believe that's what the `css` export from `styled-components` is meant to provide.